src/styles/_general/Navbar.css

```diff
@@ -7,10 +7,10 @@
   width: 100%;
   justify-content: space-between;
   padding-top: 2rem;
-  height: 7.56rem;
+  height: 5.56rem;
```

Comment:
Instead of decreasing the height of the navbar, maybe we can increase the height and add bottom padding that compensates for the height change. We can add multi pointer gradient and have the color with a higher than 0.5 alpha value (roughly 0.9 and have its position at around 75% from the top so that nav text is still visible and fading will still be there. for ex: `linear-gradient(180deg, #FFFFFF 0%, rgba(255, 255, 255, 0.9) 75.%, rgba(255, 255, 255, 0) 100%);`